ios/ReactViewPagerManager.m

```diff
@@ -34,54 +32,30 @@ - (void) goToPage
     }];
 }
 
-- (void) changeScrollEnabled
-: (nonnull NSNumber *)reactTag enabled
-: (BOOL)enabled {
-    [self.bridge.uiManager addUIBlock:^(
-                                        RCTUIManager *uiManager,
-                                        NSDictionary<NSNumber *, UIView *> *viewRegistry) {
-        ReactNativePageView *view = (ReactNativePageView *)viewRegistry[reactTag];
-        if (!view || ![view isKindOfClass:[ReactNativePageView class]]) {
-            RCTLogError(@"Cannot find ReactNativePageView with tag #%@", reactTag);
-            return;
-        }
-        [view shouldScroll:enabled];
-    }];
-}
-
-RCT_EXPORT_METHOD(setPage
-                  : (nonnull NSNumber *)reactTag index
-                  : (nonnull NSNumber *)index) {
+RCT_EXPORT_METHOD(setPage:(nonnull NSNumber *)reactTag
+                    index:(nonnull NSNumber *)index) {
     [self goToPage:reactTag index:index animated:true];
 }
 
-RCT_EXPORT_METHOD(setPageWithoutAnimation
-                  : (nonnull NSNumber *)reactTag index
-                  : (nonnull NSNumber *)index) {
+RCT_EXPORT_METHOD(setPageWithoutAnimation:(nonnull NSNumber *)reactTag
+                                    index:(nonnull NSNumber *)index) {
     [self goToPage:reactTag index:index animated:false];
 }
 
-RCT_EXPORT_METHOD(setScrollEnabled
-                  : (nonnull NSNumber *)reactTag enabled
-                  : (nonnull NSNumber *)enabled) {
-    BOOL isEnabled = [enabled boolValue];
-    [self changeScrollEnabled:reactTag enabled:isEnabled];
-}
-
-RCT_CUSTOM_VIEW_PROPERTY(scrollEnabled, BOOL, ReactNativePageView) {
-    [view shouldScroll:[RCTConvert BOOL:json]];
-}
-
-RCT_CUSTOM_VIEW_PROPERTY(keyboardDismissMode, NSString, ReactNativePageView) {
-    [view shouldDismissKeyboard:[RCTConvert NSString:json]];
-}
-
-RCT_CUSTOM_VIEW_PROPERTY(showPageIndicator, BOOL, ReactNativePageView) {
-    [view shouldShowPageIndicator:[RCTConvert BOOL:json]];
+RCT_EXPORT_METHOD(setScrollEnabled:(nonnull NSNumber *)reactTag
+                     scrollEnabled:(BOOL)scrollEnabled) {
+    [self.bridge.uiManager addUIBlock:^(RCTUIManager *uiManager, NSDictionary<NSNumber *,UIView *> *viewRegistry) {
+        ReactNativePageView *view = (ReactNativePageView *)viewRegistry[reactTag];
+        if (!view || ![view isKindOfClass:[ReactNativePageView class]]) {
+            RCTLogError(@"Cannot find ReactNativePageView with tag #%@", reactTag);
+            return;
+        }
+        [view shouldScroll:scrollEnabled];
+    }];
 }
 
 - (UIView *)view {
-    return [[ReactNativePageView alloc] initWithEventDispatcher:self.bridge.eventDispatcher];
+    return [[ReactNativePageView alloc] init];
```

Comment:
Why view is not initialized using a bridge ?

Reply:
It was a while ago when I wrote this, but I think it was because the bridge was not used.

Reply:
The bridge is used to send the gesture events to the native side